src/views/review-penalty.njk

```diff
@@ -0,0 +1,60 @@
+{% extends "layout.njk" %}
+
+{% from 'govuk/components/back-link/macro.njk' import govukBackLink %}
+{% from 'govuk/components/error-summary/macro.njk' import govukErrorSummary %}
+{% from 'govuk/components/input/macro.njk' import govukInput %}
+{% from 'govuk/components/button/macro.njk' import govukButton %}
+{% from 'govuk/components/details/macro.njk' import govukDetails %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+
+{% block pageTitle %}
+  Review your penalty details
+{% endblock %}
+{% block backLink %}
+  {{
+    govukBackLink({
+      text: 'Back',
+      href: navigation.back.href
+    })
+  }}
+{% endblock %}
+{% block main %}
+  <form method="post" action="">
+    <div class="govuk-width-container">
+      <main role="main" class="govuk-main-wrapper">
+        <div class="govuk-grid-row">
+          <div class="govuk-grid-column-two-thirds">
+            <h1 class="govuk-heading-xl">
+              <span class="govuk-visually-hidden">Review the
+            </span>Penalty details for
+            <span class="govuk-!-font-weight-bold">{{ companyName }}</span>
+            </h1>
+            <h2 class="govuk-heading-m">
+              Accounts period made up to
+              {{ madeUpToDate }}
+            </h2>
+
+            {{ 
+            govukTable({
+              caption: caption,
+              firstCellIsHeader: true,
+              head: head,
+              rows: tableRows
+            }) 
+          }}
+
+            {{ 
+            govukButton({
+              text: "Continue",
+              attributes: {
+                id: 'submit'
+              }
+            }) 
```

Comment:
The identation for govukTable and button needs to be fixed, should look something like this:

```
{{
    govukBackLink({
      text: 'Back',
      href: navigation.back.href
    })
}}
```